docs/user/include_jinja_list.md

```diff
@@ -52,7 +52,10 @@
 | is_ip_within | netutils.ip.is_ip_within |
 | is_netmask | netutils.ip.is_netmask |
 | is_network | netutils.ip.is_network |
+| is_wildcardmask | netutils.ip.is_wildcardmask |
```

Comment:
I think we need to change this to `is_reversible_wildcardmask` as `0.0.255.0` is a proper wildcard mask, just not reversible to a subnet.